### PR TITLE
Consistent use of single quotes

### PR DESCRIPTION
**Pull request by seanyoung**
We have ‘x’, `x`, and ‘x‘. Always use 'x'.

### CODE REVIEW

**Comment by LucasSte** [comment]
The first commit is already in PR #769 , isn't it?

File: src/sema/expression.rs [hunk 1]
```diff
@@ -2379,14 +2379,22 @@ fn variable(
...
             }
         }
         sym => {
-            let error = Namespace::wrong_symbol(sym, id);
-            diagnostics.push(error);
+            diagnostics.push(
+                if id.name == "now" && matches!(resolve_to, ResolveTo::Type(Type::Uint(_))) {
+                    Diagnostic::error(
+                        id.loc,
+                        "'now' was an alias for 'block.timestamp' in older version of the Solidity language. Please use 'block.timestamp' instead.".to_string(),
```
Comment (LucasSte) [inline src/sema/expression.rs:2392]: This is in another PR. Please remove.

File: tests/contract_testcases/solana/now.dot [hunk 2]
```diff
@@ -0,0 +1,26 @@
+strict digraph "tests/contract_testcases/solana/now.sol" {
```
Comment (LucasSte) [inline tests/contract_testcases/solana/now.dot:1]: This is in another PR. Please, remove.

File: tests/contract_testcases/solana/now.sol [hunk 3]
```diff
@@ -0,0 +1,12 @@
+contract c {
+	struct S { int f1; }
```
Comment (LucasSte) [inline tests/contract_testcases/solana/now.sol:2]: This is in another PR. Please, remove.

**Review by LucasSte — CHANGES_REQUESTED** [review]
All dot files have changed. I've seen some of them, because it would be unfeasible to check them all.

**Review by LucasSte — CHANGES_REQUESTED** [review]
Errors messages from the parser still contain double quotes. Can you change that?

File: src/sema/yul/tests/expression.rs [hunk 4]
```diff
@@ -1041,7 +1041,7 @@ contract testTypes {
     let ns = parse(file);
     assert!(assert_message_in_diagnostics(
         &ns.diagnostics,
-        r#"unrecognised token `:=', expected ")", ",", "address", "bool", "break", "byte", "case", "continue", "default", "for", "function", "if", "leave", "let", "return", "revert", "switch", "{", "}", identifier"#
+        r#"unrecognised token ':=', expected ")", ",", "address", "bool", "break", "byte", "case", "continue", "default", "for", "function", "if", "leave", "let", "return", "revert", "switch", "{", "}", identifier"#
```
[the remark this answers is not in the document]
Comment (seanyoung) [reply src/sema/yul/tests/expression.rs:1044]: Thanks for catching that.

[the remark this answers is not in the document]
Comment (seanyoung) [reply src/sema/yul/tests/expression.rs:1044]: See https://github.com/lalrpop/lalrpop/blob/master/lalrpop/src/lr1/codegen/parse_table.rs#L1538

[the remark this answers is not in the document]
Comment (seanyoung) [reply src/sema/yul/tests/expression.rs:1044]: Hmm it's not quite that simple. The code here https://github.com/lalrpop/lalrpop/blob/master/lalrpop/src/grammar/parse_tree.rs#L907 just using debug for escaping strings.

I think this is a problem for another day.

[the remark this answers is not in the document]
Comment (LucasSte) [reply src/sema/yul/tests/expression.rs:1044]: Ok

**Review by LucasSte — APPROVED** [review]
I haven't checked all dot files in this PR, because there were so many. I selected random one to review, but they all look good. Can you please wait for me to merge #766 before merging this PR? 
There will be many conflicts to solve.

**Comment by seanyoung** [comment]
> I haven't checked all dot files in this PR, because there were so many. I selected random one to review, but they all look good. Can you please wait for me to merge #766 before merging this PR? There will be many conflicts to solve.

Let me know when that is ready for review. I can deal with the merge conflicts,

**Pull request merged**